sys-devel/arm_none_eabi_gcc/arm_none_eabi_gcc-13.1.0.recipe

```diff
@@ -38,8 +35,8 @@ REQUIRES="
 	lib:libmpc$secondaryArchSuffix
 	"
 BUILD_REQUIRES="
-	cmd:${targetU}_as$secondaryArchSuffix >= 2.40
-	${targetU}_newlib$secondaryArchSuffix >= 4.3.0
+	cmd:${targetU}_as >= 2.40
+	${targetU}_newlib$secondaryArchSuffix >= 2.1.0
```

Comment:
I have updated newlib to version 4.3.0 and that version should be used with the new gcc. Why is this being moved back to an older version that is not in the repository anymore?